src/accelerate/test_utils/testing.py

```diff
@@ -146,6 +155,14 @@ def require_tpu(test_case):
     return unittest.skipUnless(is_tpu_available(), "test requires TPU")(test_case)
 
 
+def require_single_device(test_case):
+    """
+    Decorator marking a test that requires a single device. These tests are skipped when there are no accelerator
```

Comment:
```suggestion
    Decorator marking a test that requires a single device. These tests are skipped when there are no accelerators
```

Maybe "hardware accelerators" to disambiguate?

Reply:
`hardware accelerators` sounds more reasonable 😄